### PR TITLE
[RFR] Added .gitattributes with force unix newline conversion for *.md files

### PR DESCRIPTION
**Pull request by w0robey**
Added force newline conversion *.md files for correct work export-script on Windows system.

Specifically, for this part:

``` javascript
// bin/export   line:14

    // Not worked with Windows line endings
    // In example, '#TITLE\r\n\r\nDESCRIPTION'.match(/^.*\n+([\s\S]+)|^/)[1] is '\r', not 'DESCRIPTION'
    markdown: content.match(/^.*\n+([\s\S]+)|^/)[1]
```

### CODE REVIEW

**Comment by KittyGiraudel** [comment]
That is a good idea. :)

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+*.md text eol=lf
```
Comment (KittyGiraudel) [inline .gitattributes]: Isn’t it ironic how this file is missing EOF? :P

Comment (w0robey) [inline .gitattributes]: Actually, suprise from online editor :)

**Comment by KittyGiraudel** [comment]
Thank you! :)

**Comment by w0robey** [comment]
:+1:

**Pull request merged**